### PR TITLE
Ewl 854 add label infos to player cards

### PR DESCRIPTION
**Pull request by degouville**
https://eurosportdigital.atlassian.net/browse/EWL-854

### CODE REVIEW

File: src/modules/PlayerCard/index.js [hunk 1]
```diff
@@ -148,6 +166,7 @@ PlayerCard.propTypes = {
   className: PropTypes.string,
   onViewMoreClick: PropTypes.func,
   isViewMoreOpen: PropTypes.bool,
+  labels: labelsType.isRequired,
```
Comment (davidgarry) [inline src/modules/PlayerCard/index.js]: I'm sorry, it was not clear in the ticket and in the designs, but the labels can be different for each match. So you can't send just one instance of labels for all the matches.

File: src/modules/PlayerCard/index.js [hunk 2]
```diff
@@ -111,9 +121,17 @@ export const PlayerCard = ({
...
             forceToggle={isViewMoreOpen}
           >
-            {previousMatches.map((match, index) => (
+            {previousMatches.map(match => (
               <StyledScoreBlock key={match.id}>
-                {index > 0 && <StyledBefore />}
+                {/* NOTE: StyledBefore is dont exist in InVision's mockup */}
+                {/* NOTE: ...see https://eurosport.invisionapp.com/d/main#/console/17267787/371330695/preview */}
+                {/* NOTE: index > 0 && <StyledBefore /> */}
+                {/* TODO: Find if this is deprecated... */}
```
Comment (davidgarry) [inline src/modules/PlayerCard/index.js]: You can remove it completely, we will not use it anymore

File: src/modules/PlayerCard/index.js [hunk 3]
```diff
@@ -45,18 +46,19 @@ export const StyledScoreBlock = styled.div`
...
-    width: 19px;
-    ${breakpoints.medium(css`
-      width: 40px;
-    `)};
-    height: 20px;
-    border-right: 2px solid ${colors.coreLightMinus1};
-    display: block;
-  }
-`;
+// NOTE: Muted as not used... See L.126
```
Comment (JulienKode) [inline src/modules/PlayerCard/index.js]: Should we remove it as it is not used ?

File: src/modules/PlayerCard/mockData/playerCardMockData.js [hunk 4]
```diff
@@ -166,3 +166,15 @@ export const previousMatches = [
     },
   },
 ];
+
+export const simpleLabels = [
```
Comment (JulienKode) [inline src/modules/PlayerCard/mockData/playerCardMockData.js]: Should we add this parameter into tests ?

File: src/modules/PlayerCard/index.js (hunk 3 above)
Comment (degouville) [reply src/modules/PlayerCard/index.js]: Yes !

**Comment by degouville** [comment]
Thanks @JulienKode 😉

**Comment by degouville** [comment]
Thanks @Xb0X ;)

File: src/modules/PlayerCard/index.test.js [hunk 5]
```diff
@@ -26,4 +26,7 @@ describe('PlayerCard', () => {
     const backgroundImg = wrapper.find(StyledBackground);
     expect(backgroundImg).toHaveStyleRule('background-image', `url(${BACKGROUND_IMG})`);
   });
+
+  // TODO: it should display previousMatches
+  // TODO: it should display labels in previousMatches
```
Comment (davidgarry) [inline src/modules/PlayerCard/index.test.js]: Maybe some tests are missing ?

Comment (degouville) [reply src/modules/PlayerCard/index.test.js]: Oh yes.

Comment (degouville) [reply src/modules/PlayerCard/index.test.js]: Done  ✅

**Pull request merged**